debuglogger/default.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package debuglogger
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type defaultEvent struct {
+	level  LogLevel
+	logger *log.Logger
+	fields []byte
+}
+
+func (e *defaultEvent) Msg(msg string) {
+	if len(msg) == 0 {
+		return
+	}
+
+	e.logger.Print("[", e.level.String(), "] ", msg, string(e.fields))
+}
+
+func (e *defaultEvent) Str(key, val string) Event {
+	e.fields = append(e.fields, ' ')
+	e.fields = append(e.fields, key...)
+	e.fields = append(e.fields, '=')
+	if strings.Contains(val, " ") {
+		e.fields = append(e.fields, strconv.Quote(val)...)
+	} else {
+		e.fields = append(e.fields, val...)
+	}
+
+	return e
+}
+
+func (e *defaultEvent) Err(err error) Event {
+	if err == nil {
+		return e
+	}
+
+	e.fields = append(e.fields, " error=\""...)
+	e.fields = append(e.fields, err.Error()...)
+	e.fields = append(e.fields, '"')
+	return e
+}
+
+func (e *defaultEvent) Errs(errs ...error) Event {
```

Comment:
Not sure if I would add any multi versions for now, especially since it's only one of the types so inconsistent

Reply:
I added it because of https://github.com/corazawaf/coraza/pull/682/files/3b216855c045a339da6e4913ff6e32d486b51f75#diff-32bf0985de09d747ed6a6d8400fa4e02e61e15b9cb67b3da4623c050bf1306ccR227

Reply:
If it's just one spot let's go ahead and loop there for now I think, doesn't pass the "rule of three" :P

Reply:
Makes sense.

Reply:
Removed.